multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -499,7 +499,7 @@ def transform_image(
             if col_type == ROIS:
                 processed_data = df[col_name].tolist()
             elif col_type in [IMAGE_PATH, IMAGE]:
-                processed_data = col_value.apply(lambda ele: ele.split(";")).tolist()
+                processed_data = col_value.apply(lambda ele: str(ele).split(";")).tolist()
```

Comment:
This will convert the None value to ["None"], will that introduce issue during training as the image can't be loaded or we will just ignore invalid path later one

Reply:
It will be guarded by https://github.com/autogluon/autogluon/blob/70dd543682a6ed0bb1173274e5de63071a71336e/multimodal/src/autogluon/multimodal/data/process_image.py#L359-L365

Reply:
We would try open the images and skip them if failures happen.